preview/arcus-general.md

```diff
@@ -0,0 +1,71 @@
+# Arcus - Azure development in a breeze.
+
+## What is Arcus?
+A set of Codit-developed OSS libraries/tools that simplify building apps running on Microsoft Azure.
+It is Codit’s strategy to centralize company-wide best practices and use it across all our customers.
+
+* Built By Experts, For Experts
+  Sponsored by Codit! Maintained by a dedicated set of Codit experts that assures quality for our current and future clients.
+
+* Modular Design
+  All Arcus components are highly functional and composable. They can be used separately, but they are better together.
+
+* Open-Source & Free to use
+  The Arcus framework is constantly updated by contributions from Codit employees and abroad!
+  It is free to use under the MIT license.
+
+🚩 Are you doing the same thing at multiple customers? Then we are happy to hear! [Link to arcus GitHub]()
+
+## Why should you use Arcus?
+There exists many reasons to use Arcus in your current and future projects.
+
+* Time-saving
+  Arcus contains tremendous amounts of boilerplate code that helps you save loads of time during project development.
+  It helps with infrastructure like secure secret management, application-wide telemetry tracking, tedious DevOps operations. and even has fully development-ready project templates to get you started immediately.
+
+* Centralized
+  The entire Arcus framework is maintained and tested the same way, with the same set of dedicated experts. There exists a single point of truth were everything is maintained and made ready for your benefit.
+
+* Best practices
+  Every Arcus component has the company-wide best practices backed right into the code. It is secure by-default, tracks telemetry by itself, automates hard-to-setup DevOps tasks, and many more.
+
+## How to use Arcus?
```

Comment:
The list below is more of a `When to use Arcus?` instead of `How to use Arcus?`
Maybe we should also mention this is not a complete list and are examples of usage. For example Arcus.Scripting doesn't necessarily have to be used from a DevOps pipeline 😉

Reply:
Yes, you're right. Was experimenting with this as I wanted something to get people started after they are convinced by the previous two titles 😅 . I think the 'How to use it' is till relevant but we should maybe change the structure of this. Maybe more focus on the project templates as they are the starting point. But I'm thinking how we will introduce ML and Scripting to this.

And yeah, we should make sure that we mention that this is by all means not 'complete'.

Reply:
I think the current setup is a good one, `What?` to `Why?` to `How?` seems like a good walk-through 👍🏻 
Or maybe use `How and When?`?